core/src/main/java/org/opentosca/toscana/core/transformation/UnsupportedPlatformException.java

```diff
@@ -0,0 +1,19 @@
+package org.opentosca.toscana.core.transformation;
+
+import org.opentosca.toscana.core.transformation.platform.Platform;
+
+/**
+ * Gets thrown when a platform is encountered which is not supported by any known plugin
+ */
+public class UnsupportedPlatformException extends Exception {
```

Comment:
This is just a sidenote:
If  this has to be handled by the API: add `@ResponseStatus(value = <Error Code> = <Some message>)` here to tell spring what to respond when this exception gets thrown